absolventenfeier/einladung.tex

```diff
@@ -0,0 +1,130 @@
+\documentclass[parskip=half, fontsize=10pt, paper=a5]{scrartcl}
+
+\usepackage[margin=0.5cm, left=3cm]{geometry}
+
+\usepackage{fontspec}
+\setmainfont{Akkurat Office}
+
+\usepackage[ngerman]{babel}
+
+\usepackage{csquotes}
+\usepackage{fontawesome5}
+
+\usepackage{graphicx}
+\usepackage{enumitem}
+% \setlist[description]{leftmargin=0.5em}
+
+\pagestyle{empty}
+
+\usepackage{tikz}
+
+\usepackage{xcolor}
+\definecolor{tu}{RGB}{131,184,23}
+\definecolor{physik}{RGB}{8,81,162}
+
+\usepackage[colorlinks,urlcolor=tu]{hyperref}
+
+
+\begin{document}
+\raggedright
+
+\begin{tikzpicture}[remember picture, overlay, shift=(current page.north west)]
+  \fill[tu] (0.5cm, 0) rectangle (2.5cm, -\paperheight);
+  \node[anchor=north west, inner sep=0.1cm] at (0.5cm, -0.5cm) {%
+    \includegraphics[width=1.8cm]{schwingung_positiv_white.pdf}
+  };
+\end{tikzpicture}
+
+\begin{center}
+\textbf{\Large Einladung zur Absolventenfeier\\ der Jahrgänge 2019 – 2022}
+\end{center}
+\vspace{0.5cm}
+
+Sehr geehrte Absolventinnen und Absolventen,
+
+zu Ihrem in den Jahren 2019 – 2022 an der TU Dortmund erworbenem Abschluss möchten
+wir Ihnen ganz herzlich gratulieren.
+Die Fakultät Physik under der Alumni-Verein PeP et al.\ e.\,V.\ sind der Meinung, dass dies ein Grund zum Feiern ist.
+Daher möchten wir Sie und Ihre Angehörigen zur diesjährigen Absolventenfeier einladen.
+
+\vspace{0.25cm}
+\textcolor{tu}{\textbf{\large Informationen}}
+\begin{description}[style=multiline, leftmargin=5em]
+  \item[Wann] 1. April 2023, 15:00 Uhr
+  \item[Wo] Campus Nord\\
+    \textbf{Audimax}\\
+    Vogelpothsweg 87, 44227 Dortmund
+\end{description}
+
+\vspace{0.25cm}
+\textcolor{tu}{\textbf{\large Anmeldung}}
+
+Wenn Sie an der Absolventenfeier teilnehmen möchten, melden Sie sich dazu bitte unter
+\begin{center}
+  \large
+  \href{https://registration.pep-dortmund.org/events/absol23/registration}{registration.pep-dortmund.org} 
+\end{center}
+an.
+
+Wir bitten um Ihr Verständnis, dass auf Grund der großen Anzahl an erwarteten Teilnehmenden nur zwei weitere
+Gäste mitgebracht werden können.
+
+Die Veranstaltung beginnt um 16:00 Uhr. Vorher möchten wir gemeinsam mit allen Absolvent*innen ein Gruppenfoto aufnehmen.
```

Comment:
Maybe people stop reading here and will arrive to late?

Reply:
People who stop reading after the first sentence deserve not to be on the group photo maybe? 🤣